app_mobilenet_alpha_1/build_mobilenet_alpha_1.py

```diff
@@ -0,0 +1,96 @@
+from xmos_ai_tools import xformer as xf
+
+import tensorflow as tf
+from tensorflow.keras.applications.mobilenet_v2 import MobileNetV2
+
+import os
+
+input_shape = (160, 160, 3)
+alpha_value = 1.0
+model_weights = None
+classes = 16
+classifier_activation = None
+
+# Create MobileNet model
+model = MobileNetV2(
+    input_shape=input_shape,
+    alpha=alpha_value,
+    weights=model_weights,
+    classes=classes,
+    classifier_activation=classifier_activation,
+)
+
+# Use tf lite onverter to quantize the model to int8
+converter = tf.lite.TFLiteConverter.from_keras_model(model)
+
+# Set the optimization mode
+converter.optimizations = [tf.lite.Optimize.DEFAULT]
+
+
+# Use a representative dataset generator for proper INT8 optimization
+def representative_dataset_gen():
+    for _ in range(100):
+        yield [tf.random.normal([1] + list(input_shape))]
+
+
+converter.representative_dataset = representative_dataset_gen
+
+# Ensure that if any ops can't be quantized, the converter throws an error
+converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
+
+converter.inference_input_type = tf.int8
+converter.inference_output_type = tf.int8  #
+
+# quantize the model
+tflite_model = converter.convert()
+
+# Save the quantized model
+with open("mobilenetV2_input_160x160x3_alpha_1.tflite", "wb") as f:
+    f.write(tflite_model)
+
+print("Model quantized and saved successfully.")
+
+# split the model
+xf.convert(
```

Comment:
Could you please change this so that we only need to invoke xformer once?

I had tried something like this for MobileNetV2, 

`./bazel-bin/xcore-opt model_maker/pretrainedmodel.tflite -o model.tflite --xcore-op-split-tensor-arena=1 --xcore-op-split-top-op=8,18 --xcore-op-split-bottom-op=16,22 --xcore-op-split-num-splits=10,4 --xcore-conv-err-threshold=3.6 --xcore-offline-offsets=1 --xcore-overlap=1 --xcore-tflmc-print --xcore-flash-image-file=1.params`